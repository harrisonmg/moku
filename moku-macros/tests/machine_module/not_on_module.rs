extern crate moku_macros as moku;

use moku::*;

#[state_machine]
mod blinky {
    use moku::*;

    #[machine_module]
    struct Module {}
}

fn main() {}
