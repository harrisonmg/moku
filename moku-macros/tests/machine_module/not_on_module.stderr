error: expected `mod`
  --> tests/machine_module/not_on_module.rs:10:5
   |
10 |     struct Module {}
   |     ^^^^^^
